packages/bpk-component-chip/src/BpkDropdownChip.tsx

```diff
@@ -16,36 +16,35 @@
  * limitations under the License.
  */
 
-/* @flow strict */
-
 /*
 The dropdown chip component is just a selectable chip component
 with a trailing accessory view of a chevron down icon.
 */
 
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
 import ChevronDownIconSm from '../../bpk-component-icon/sm/chevron-down';
 
 import BpkSelectableChip from './BpkSelectableChip';
-import {
-  COMMON_PROP_TYPES,
-  COMMON_DEFAULT_PROPS,
-  type CommonProps,
-} from './commonTypes';
-
-export type Props = CommonProps;
-
-const BpkDropdownChip = (props: Props) => {
-  const { ...rest } = props;
-
-  return (
-    <BpkSelectableChip
-      {...rest}
-      trailingAccessoryView={<ChevronDownIconSm />}
-    />
-  );
-};
-
-BpkDropdownChip.propTypes = COMMON_PROP_TYPES;
-BpkDropdownChip.defaultProps = COMMON_DEFAULT_PROPS;
+import type { CommonProps } from './commonTypes';
+import { CHIP_TYPES } from './commonTypes';
+
+export interface Props extends CommonProps {}
```

Comment:
Same thing here as above it doesn't seem relevant/required given we don't add to it